### ISSUE TITLE
HmIP-STHD exposes less actions than HmIP-WTH-2

### ISSUE THREAD

**Issue by yschroeder** [danielperna84/pyhomematic#338]
**Home Assistant version (if applicable):**
0.115.6

**Describe the bug**
I have two types of wall thermostats `HmIP-STHD` and `HmIP-WTH-2`. Both are used in different rooms as the "brain" to control temperatures. The only visible difference is that `HmIP-WTH-2` has a control knob, and the `HmIP-STHD` doesn't. However, in Home Assistant they behave differently. The WTH-2 shows 3 buttons in Home Assistant (`auto`, `heat`, `off`). The STHD only shows `heat` and `off`. Also it displays some error message when clicking those two buttons:

```
2020-10-04 20:07:01 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection.2958768904] %i format: a number is required, not NoneType
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 137, in handle_call_service
    await hass.services.async_call(
  File "/usr/src/homeassistant/homeassistant/core.py", line 1315, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1350, in _execute_service
    await handler.func(service_call)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 204, in handle_service
    await self.hass.helpers.service.entity_service_call(
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 459, in entity_service_call
    future.result()  # pop exception if have
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 655, in async_request_call
    await coro
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 490, in _handle_entity_call
    await result
  File "/usr/src/homeassistant/homeassistant/components/climate/__init__.py", line 430, in async_set_hvac_mode
    await self.hass.async_add_executor_job(self.set_hvac_mode, hvac_mode)
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/homeassistant/homeassistant/components/homematic/climate.py", line 152, in set_hvac_mode
    self._hmdevice.turnoff()
  File "/usr/local/lib/python3.8/site-packages/pyhomematic/devicetypes/thermostats.py", line 321, in turnoff
    self.actionNodeData('CONTROL_MODE', self.MANU_MODE)
  File "/usr/local/lib/python3.8/site-packages/pyhomematic/devicetypes/generic.py", line 349, in actionNodeData
    return self._setNodeData(name, self.ACTIONNODE, data, channel)
  File "/usr/local/lib/python3.8/site-packages/pyhomematic/devicetypes/generic.py", line 363, in _setNodeData
    LOG.error("HMDevice.setNodeData: %s not found with value %s on %i" %
TypeError: %i format: a number is required, not NoneType
```

**To Reproduce**
1. Add thermostat control to Lovelace UI for `HmIP-STHD` device
2. Click the `heat` or `off` button
3. Error message shows up
4. `auto` button is missing

**Expected behavior**
I would expect that I can set the STHD to `auto` mode the same way I can do this in the CCU web interface. Also it should not show errors when using the other buttons. I would expect both devices to work identically, as they also work identically in CCU (exactly the same configuration options). Internally both devices even use the same PCB. The STHD just has the rotary encoder not populated.

**Screenshots**
 `HmIP-WTH-2` has `auto` button and also shows automatic mode correctly:
![screenshot_wth2](https://user-images.githubusercontent.com/8067560/95025476-43579980-068a-11eb-8bfd-9a6cb62e61ae.PNG)

 `HmIP-STHD` has no `auto` button, shows `heat` mode while it is actually in `auto` mode according to CCU.
![screenshot_sthd](https://user-images.githubusercontent.com/8067560/95025496-62562b80-068a-11eb-9755-c78b70c5dd42.PNG)

**Additional context**
CCU Version: 3.51.6 running on a Pi via PiVCCU3

I looked through the code and without understanding it fully I would think that the correct actions are missing in this line: https://github.com/danielperna84/pyhomematic/blob/483e593d4ce29598b10dbccbe8c5b2b5fdab53d8/pyhomematic/devicetypes/thermostats.py#L299

**Comment by yschroeder** [comment]
Is there anything I can do to help here?

**Comment by yschroeder** [comment]
I changed the type of the `HmIP-STHD` from `IPThermostatWall` to `IPThermostatWall2` after carefully comparing the output of `get_device_data.py`. The output is nearly identical (as expected). Now, the  `HmIP-STHD` behaves correctly. I can switch between automatic and manual mode and turn it off.

https://github.com/danielperna84/pyhomematic/blob/b771ab8a0022ea4e6011fe433860a2b260a44891/pyhomematic/devicetypes/thermostats.py#L461

I will probably create a PR to fix this.

**Comment by yschroeder** [comment]
This is fixed in #395.
